task/BuildOTPGraph.js

```diff
@@ -21,7 +22,19 @@ const buildGraph = function (router) {
     );
     const commit = version.toString().match(/commit: ([0-9a-f]+)/)[1];
 
-    const command = `docker run -e JAVA_OPTS="${JAVA_OPTS}" -v ${dataDir}/build/${router.id}:/var/opentripplanner --mount type=bind,source=${dataDir}/../logback-include-extensions.xml,target=/logback-include-extensions.xml ${dockerImage} --build --save`;
+    let command = null;
```

Comment:
```suggestion
    let command;
```

Reply:
I think = null is more readable, but I removed it

Reply:
Null isn't ever used here so it kinda gives an image that the value could have null when it can't.